api/lib/libDependencies.js

```diff
@@ -203,6 +205,15 @@ const jigsawFetchRecordsGateway = require('./gateways/Jigsaw/FetchRecord')({
   logger
 });
 
+/*const cautionaryAlertsGateway = require('../../../lib/gateways/CautionaryAlerts/CautionaryAlertsApi')(
```

Comment:
Is this commented because it's not used yet? Just checking. I'm fine with it if that's the case as it's useful to help people when they come to use it

Reply:
Yeah, just added for future use.